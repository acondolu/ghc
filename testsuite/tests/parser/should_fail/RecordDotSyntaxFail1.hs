{-# LANGUAGE RecordDotSyntax #-}

no Foo { bar.baz = x } = undefined
  -- Syntax error: "Field selector syntax is not supported in
  -- patterns."
