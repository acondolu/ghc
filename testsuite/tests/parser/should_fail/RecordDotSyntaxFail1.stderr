RecordDotSyntaxFail1.hs:3:10:
    Field selector syntax is not supported in patterns.
